data/channels.csv

```diff
@@ -28929,4 +28930,4 @@ Zvezda.ru,Zvezda,,,RU,,,c/RU,rus,general,FALSE,,,,https://i.imgur.com/n9zLVyR.pn
 ZvezdaPlus.ru,Zvezda Plus,,,RU,,,c/RU,rus,,FALSE,,,https://tvzvezda.ru/sp/zvezda-plus/,https://cdn.red-media.ru/wp-content/uploads/2021/02/Star_PLUS.png
 ZvezdaTV.rs,Zvezda TV,,,RS,,,c/RS,srp,,FALSE,,,,https://mts.rs/oec/images/tv_channels/8cff6095819ea749b6580f73c7d31d26.jpg
 ZVTAVS.nl,ZVT-AVS,,,NL,,,c/NL,nld,,FALSE,,,,https://i.imgur.com/9PG10sW.png
-Zwei2MusicTV.de,Zwei 2 Music TV,,,DE,,,c/DE,deu,music,FALSE,,,https://folx.network/zwei-music-television/,https://i.imgur.com/VVwHID9.png
\ No newline at end of file
+Zwei2MusicTV.de,Zwei 2 Music TV,,,DE,,,c/DE,deu,music,FALSE,,,https://folx.network/zwei-music-television/,https://i.imgur.com/VVwHID9.png

```

Comment:
```suggestion
Zwei2MusicTV.de,Zwei 2 Music TV,,,DE,,,c/DE,deu,music,FALSE,,,https://folx.network/zwei-music-television/,https://i.imgur.com/VVwHID9.png
```

Reply:
The text editor does not allow me to save without spaces. So it's a problem unrelated to mine. Possibly, I will try to correct it in the future.

Reply:
I have tried the website, but it won't let me save without the spaces in your suggestion.